.github/workflows/sql-test-workflow.yml

```diff
@@ -2,6 +2,13 @@ name: SQL Plugin Tests
 
 on:
   workflow_dispatch:
+    inputs:
+      name:
```

Comment:
can we call this `branch_commit` or similar?

Reply:
That's a custom run name which could be specified to override the default value. Default is `<branch name> @ <commit sha>`